### DISCUSSION TITLE
Aditional settings (Advanced tab)

### DISCUSSION THREAD

**Discussion by Cvetaca** [NginxProxyManager/nginx-proxy-manager#3745] [General]
I understand that there is an "Advanced" tab in configuration options for reverse proxy. But I don't know the syntax I have to use there. For example if I want to add this config:
```
location / {
        proxy_http_version 1.1;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header Host $http_host;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection $connection_upgrade_keepalive;
    }
```
Do I write it like this inside this field? If I do use this field do other settings in other windows get overriden? Is is possible to just append extra things on it?
Thank you,
Tjaž

**Comment by SirKubiac** [comment]
Some kind of manual would be nice. Or a link to a manual, if it already exists.
